docs/docs/getting-started-google-compute-enging/index.md

```diff
@@ -0,0 +1,137 @@
+# Getting Started with Google Compute Engine
+
+Getting started with GCE requires a few 
+
+1. **[Sign Up](#sign-up)**
+2. **[Start a Project](#start-project)**
+3. **[Create your Service Account Key](#create-key)**
+4. **[Create your Stack](#create-stack)**
+5. **[Set your Credentials](#set-creds)**
+6. **[Save your Stack and Initialize](#save-init)**
+
+***
+
+## Sign Up <a name="sign-up"></a>
+
+>Using GCE requires you to sign up for the Google Cloud Platform. This will also require you to provide personal information and a credit card. You'll be given a 60 day free trial and $300 in credit. Also, you won't be charged anything until you are once again prompted to do so. [Learn more about pricing](https://cloud.google.com/compute/pricing)
+
+Sign up for Google Cloud Platform by going to [https://cloud.google.com/](https://cloud.google.com/). Once there, click 'Try It Free'. This will take you to a sign up wizard. Go through it (select your country, agree to terms, etc). At this point you'll be prompted for your personal information and a credit card. Once you offer all that, you'll be signed up and signed in!
+
+> Once you are signed up, you will be prompted to Try Compute Engine, and will be offered a 15 minute quickstart. It might be a good idea to go through this process to get a solid understanding of the platform.
+
+***
+
+## Start a Project <a name="start-project"></a>
+
+Google Cloud Platform starts you out in a default 'project' called My First Project. Projects are just a way of organizing your resources to be associated with the project they are used for. 
+
+You might not want to stick with this default project, so go ahead and create a new one:
+
+1. From the Dashboard or Home screen, you'll see a drop-down at the top of the screen with the name of your project displayed (defaulted to My First Project). Click that to open the menu.
+2. Click Create Project.
+3. This will open a dialog screen for you to name your project. Click Show Advanced Options to select your Region. This is important even if you don't change the region, because you'll be asked for the Region later when setting up credentials for your Stack.
+
+> Take note not only of the Project Name but the Project ID. You will need this ID when creating credentials for your Koding Stack. 
```

Comment:
I would emphasize that they should write down or copy the project ID somewhere.  Taking note of it doesn't mean they'll remember it.